FirebaseRemoteConfig/Tests/Unit/RCNPersonalizationTest.m

```diff
@@ -116,14 +116,15 @@ - (void)setUp {
   RCNConfigContent *configContent = [[RCNConfigContent alloc] initWithDBManager:DBManager];
 
   // Create a mock FIRRemoteConfig instance.
-  _configInstance = OCMPartialMock([[FIRRemoteConfig alloc]
-      initWithAppName:@"testApp"
-           FIROptions:[[FIROptions alloc] initWithGoogleAppID:@"1:123:ios:test"
-                                                  GCMSenderID:@"testSender"]
-            namespace:@"namespace"
-            DBManager:DBManager
-        configContent:configContent
-            analytics:_analyticsMock]);
+  FIROptions *options = [[FIROptions alloc] initWithGoogleAppID:@"1:123:ios:test"
```

Comment:
It's now fatal not to have an api Key - which should make detection of an invalid app earlier